package.json

```diff
@@ -1,42 +1,48 @@
 {
-  "name": "nemo-screenshot",
```

Comment:
This indentation change seems unnecessary and actually makes it different than other nemo projects.